charmcraft/commands/store/__init__.py

```diff
@@ -583,3 +608,54 @@ def _get_lib_info(*, full_name=None, lib_path=None):
     return LibData(
         lib_id=libid, api=libapi, patch=libpatch, content_hash=content_hash, content=content,
         full_name=full_name, path=lib_path, lib_name=lib_name, charm_name=charm_name)
+
+
+class CreateLibCommand(BaseCommand):
+    """Create a charm library."""
+    name = 'create-lib'
+    help_msg = "Create a charm library."
+    overview = textwrap.dedent("""
+        Create a charm library.
+
+        It will request a unique ID from Charmhub and bootstrap a
+        template file in the proper local directory.
+
+        It will automatically take you through the login process if
+        your credentials are missing or too old.
```

Comment:
```suggestion
        You will be interactively queried for your login credentials if they
        are missing or if they have expired.
```

Reply:
I think this improvement is nice, but won't apply it here for consistency (this message appears in a lot of commands, so we would better change it in all at the same time). If you think that's worthwhile, please open an issue for it, thanks!